cmd/style/style.go

```diff
@@ -0,0 +1,113 @@
+package style
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/charmbracelet/lipgloss"
+)
+
+const (
+	// White represents the common white color used throughout the CLI.
+	White = "252"
```

Comment:
I assume this is RGB. Why not `255` which would be a clean bright white? Is it to highlight the color?

Reply:
Reducing contrast, mainly. I haven't checked it on an ubuntu themed terminal with a purple background yet though.

Reply:
Yeah the white on the default terminal isn't too bright. But good question if it's even a 255.